google/cloud/dataproc_spark_connect/exceptions.py

```diff
@@ -24,4 +79,4 @@ def __init__(self, message):
         super().__init__(message)
 
     def _render_traceback_(self):
-        return self.message
+        return [self.message]
```

Comment:
I think that we should apply this change only to non-Colab/Colab Enterprise IDEs. May you use helpers in https://github.com/GoogleCloudDataproc/dataproc-spark-connect-python/blob/main/google/cloud/dataproc_spark_connect/environment.py for this?